applications/tari_merge_mining_proxy/src/main.rs

```diff
@@ -49,7 +49,7 @@ async fn main() -> Result<(), MmProxyError> {
     let config = initialize()?;
 
     let addr = config.proxy_host_address;
-    println!("Listening on {}...", addr);
+    println!("\nListening on {}...\n", addr);
```

Comment:
nit: This should be replaced with a log probably?

Reply:
This is intended user feedback